browser/src/control/Control.NotebookbarWriter.js

```diff
@@ -2169,6 +2169,7 @@ window.L.Control.NotebookbarWriter = window.L.Control.Notebookbar.extend({
 	},
 
 	getReviewTab: function() {
+		var hideChangeTrackingControls = this._map['wopi'].HideChangeTrackingControls;
```

Comment:
Nit: new code is better to use 'let' for mutable variables and 'const' for variables that you don't re-assign; 'var' has to behave in an odd way for backwards compatibility.